dev-util/kdstatemachineeditor/kdstatemachineeditor-1.2.4.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils kde5-functions
```

Comment:
The use of kde5-functions seems to be not warranted here if it only is for `add_qt_dep` function. Please drop it and replace with regular `category/pn:slot[use]` atoms.

Reply:
All your `dev-qt/` dependencies need to be fixed to `slot 5` (maybe I was not clear enough above).